docs/docs/rest-api/public/api/v2/examples/queue.json

```diff
@@ -145,9 +145,9 @@
             "processed": 23
           },
           {
-            "reason": "InsufficientGpus",
+            "reason": "DeclinedScarceResources",
```

Comment:
Should the examples with `InsufficientGpus` be kept instead of being deleted?

Reply:
they were in the example two times before this change ... I actually wondered the same when reviewing :)

Reply:
yep, it seems it was a copy paste error before